bin/pt-show-grants

```diff
@@ -2433,6 +2467,16 @@ type: array
 
 Only show grants for this comma-separated list of users.
 
+=item --convert-MariaDB
+
+When set it convert some of the proprietary MariaDB syntax into valid MySQL form
```

Comment:
Can't we do this automatically if we connect to MariaDB? Or will it break for valid MySQL syntax?

Reply:
Looks like not, useful option.